### PR TITLE
[4.1] configuration.php-dist

### PR DESCRIPTION
**Pull request by brianteeman**
Example settings were missing from this file

Code review only

### CODE REVIEW

**Comment by zero-24** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37040">issues.joomla.org/tracker/joomla-cms/37040</a>.</sub>

File: installation/configuration.php-dist [hunk 1]
```diff
@@ -142,4 +158,6 @@ class JConfig
 
 	/* Miscellaneous Settings */
 	public $asset_id = 1;
+	public $behind_loadbalancer = false;
+	public $block_floc = 1;
```
Comment (heelc29) [inline installation/configuration.php-dist]: Is this still needed? #36855

Comment (zero-24) [reply installation/configuration.php-dist]:
```suggestion
```

nope

**Comment by brianteeman** [comment]
I am closing this. @zero-24 you already refused to merge the floc change. I do not appreciate you making a change to my PR

**Comment by zero-24** [comment]
> @zero-24 you already refused to merge the floc change.

No that is not true the PR is just awaiting the 4.1.0 and 3.10.6 release which this PR would have been waiting for too.

>  I do not appreciate you making a change to my PR

My intention was to help you and not bother you with obvius changes. When you dont want maintainers to be allowed to do changes you can uncheck the option when creating the PR ;)

**Comment by zero-24** [comment]
New PR done here: https://github.com/joomla/joomla-cms/pull/37041

**Pull request closed without merging**